src/Etg.Yams/YamsEntryPoint.cs

```diff
@@ -12,24 +12,24 @@ public class YamsEntryPoint
     {
         private readonly IDeploymentWatcher _deploymentWatcher;
         private readonly IApplicationPool _applicationPool;
-        private static IUnityContainer _unityContainer;
+        private static IContainer _container;
 
         public YamsEntryPoint(YamsConfig config) : this(InitializeDefaultUnityContainer(config))
         {
         }
 
-        public YamsEntryPoint(IUnityContainer unityContainer)
+        public YamsEntryPoint(IContainer container)
         {
-            _deploymentWatcher = unityContainer.Resolve<IDeploymentWatcher>();
-            _applicationPool = unityContainer.Resolve<IApplicationPool>();
-            _unityContainer = unityContainer;
+            _deploymentWatcher = container.Resolve<IDeploymentWatcher>();
+            _applicationPool = container.Resolve<IApplicationPool>();
+            _container = container;
         }
 
-        public static IUnityContainer InitializeDefaultUnityContainer(YamsConfig config)
+        public static IContainer InitializeDefaultUnityContainer(YamsConfig config)
```

Comment:
You forgot the *Unity* here :)

`InitializeDefaultModules` is probably a better name for this method?